Include/internal/pycore_object.h

```diff
@@ -320,7 +320,7 @@ static inline void _PyObject_GC_TRACK(
                           "object already tracked by the garbage collector",
                           filename, lineno, __func__);
 #ifdef Py_GIL_DISABLED
-    op->ob_gc_bits |= _PyGC_BITS_TRACKED;
+    _PyObject_SET_GC_BITS(op, _PyGC_BITS_TRACKED);
```

Comment:
We want to avoid expensive atomic operations when initializing objects or tracking/untracking objects. These will have a significant impact on performance.

Reply:
Sounds good. Let me see if that's workable.